Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:erbium
```

Comment:
@jackvanb I looked up node:erbium and it was last updated 7 years ago in 2014... is that the right image to pull from?

Reply:
https://nodejs.org/en/about/releases/ - I believe Erbium is the stable version of Node 12 that was released in 2019 and is maintained actively until 2022. I can also try it with the stable version of Node 14 to get the latest.

Reply:
@jackvanb Locally I have been running NodeJS 14, which is the current LTS. Let's stick with the current LTS version